src/transforms/transform-code-scanning-alert.ts

```diff
@@ -101,3 +109,43 @@ export const transformCodeScanningAlert = async (context: WebhookContext, github
 		}]
 	};
 };
+
+export const transformCodeScanningAlertToJiraSecurity = async (context: WebhookContext, githubInstallationId: number, jiraHost: string): Promise<JiraVulnerabilityBulkSubmitData | null> => {
+	const { alert, repository } = context.payload;
+
+	if (!alert.most_recent_instance?.ref?.startsWith("refs/heads")) {
+		context.log.info("Skipping code scanning alert detected on a pull request.");
+		return null;
+	}
+
+	const metrics = {
+		trigger: "webhook",
+		subTrigger: "code_scanning_alert"
+	};
+	const gitHubInstallationClient = await createInstallationClient(githubInstallationId, jiraHost, metrics, context.log, context.gitHubAppConfig?.gitHubAppId);
+
+	const handleUnmapped = (state) => context.log.info(`Received unmapped state from code_scanning_alert webhook: ${state}`);
```

Comment:
Can we also add a `field` param here so we can tell if it was the `severity` or `status` that received the unmapped value.